src/isNonPositive.js

```diff
@@ -0,0 +1,30 @@
+import { both, lte, flip } from 'ramda';
+
+import isNumber from './isNumber';
+
+/**
+ * Checks if value is a non-positive `Number` primitive or object. This includes all negative
+ * numbers and zero.
+ *
+ * @func isNonPositive
+ * @memberOf RA
+ * @since {@link https://char0n.github.io/ramda-adjunct/1.15.0|v1.15.0}
+ * @category Type
+ * @sig * -> Boolean
+ * @param {*} val The value to test
+ * @return {Boolean}
+ * @see {@link RA.isNegative|isNegative}
```

Comment:
`@see {@link RA.isNegative|isNegative}, {@link RA.isNonNegative|isNonNegative}`

Also I would add the following into the `isPositive` source:
`@see {@link RA.isNonPositive|isNonPositive}`